scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -347,27 +341,25 @@ def positive(w): return '{:,}'.format(np.sum(w > 0))
 predict_df['probability_str'] = predict_df['probability'].apply('{:.1%}'.format)
 
 
-# In[25]:
+# In[24]:
 
 # Top predictions amongst negatives (potential hidden responders)
 model = 'full'
-predict_df.sort_values('decision_function', ascending=False).    query("status == 0 & feature_set == '{0}'".format(model)).head(10)
+(predict_df
+    .sort_values('decision_function', ascending=False)
+    .query("status == 0 and feature_set == @model")
+    .head(10)
+)
 
 
-# In[26]:
+# In[25]:
 
 # Ignore numpy warning caused by seaborn
 warnings.filterwarnings('ignore', 'using a non-integer number instead of an integer')
```

Comment:
Check if this is still needed. I think it may be able to be removed due to environment updates.